CHANGELOG.md

```diff
@@ -0,0 +1,163 @@
+- The definition of `Field` has been altered. It is no longer necessary to provide an instance for this, there is a single instance now for every type that implements `EuclideanRing` and `DivisionRing` .
+
+**Other changes**
+
+- `Monoid` has been moved into the prelude
+- `Data.Symbol` has been moved into the prelude
+- `RProxy` and `RLProxy` have been moved into the prelude
+- There are now various instances available for records: `Eq`, `Show`, `Semigroup`, `Monoid`, `Semiring`... etc. (@i-am-tom)
+
+## [v3.3.0](https://github.com/purescript/purescript-prelude/releases/tag/v3.3.0) - 2018-04-13
+
+- Added `Data.Function.applyN` for repeatedly applying a function to an initial value (@matthewleon)
+
+## [v3.2.0](https://github.com/purescript/purescript-prelude/releases/tag/v3.2.0) - 2018-04-07
+
+- Added `Bounded` instance for `Number` (@erisco)
+
+## [v3.1.1](https://github.com/purescript/purescript-prelude/releases/tag/v3.1.1) - 2017-11-01
+
+Remove unused `refIneq` function (@metaleap)
+
+## [v3.1.0](https://github.com/purescript/purescript-prelude/releases/tag/v3.1.0) - 2017-06-27
+
+* Fixes for out-of-date documentation (@joneshf, @matthewleon)
+* Add a new `DivisionRing` class
+
+## [v3.0.0](https://github.com/purescript/purescript-prelude/releases/tag/v3.0.0) - 2017-03-25
+
+- Updated for PureScript 0.11
+- Removed incorrect `EuclideanRing Unit` and `Field Unit` instances
+
+## [v2.5.0](https://github.com/purescript/purescript-prelude/releases/tag/v2.5.0) - 2017-03-02
+
+- Added `Eq1` and `Ord1` classes
+
+## [v2.4.0](https://github.com/purescript/purescript-prelude/releases/tag/v2.4.0) - 2017-02-13
+
+- Add the new `Discard` class to support https://github.com/purescript/purescript/pull/2653
+- Fixes a corner case in the `degree` function for `Int`.
+- Documentation fixes
+
+## [v2.3.0](https://github.com/purescript/purescript-prelude/releases/tag/v2.3.0) - 2017-01-29
+
+- Added `gcd` and `lcm` for any `EuclideanRing`.
+- Fixed some issues with the `EuclideanRing` laws. The `Unit` instance is no longer valid, but is being kept around temporarily for the sake of backwards compatibility.
+
+## [v2.2.0](https://github.com/purescript/purescript-prelude/releases/tag/v2.2.0) - 2017-01-26
+
+- Added instances of the form `C b => C (a -> b)` for `Semiring`, `Ring`, `CommutativeRing`, `BooleanAlgebra`
+- Added some documentation for the `Void` type (@chexxor)
+- Fixed documentation typos (@mlang, @thoradam)
+
+## [v2.1.0](https://github.com/purescript/purescript-prelude/releases/tag/v2.1.0) - 2016-09-30
+
+- Added `whenM` and `unlessM` variants of `when` and `unless` for situations where the conditional is also in `m`
+
+## [v2.0.0](https://github.com/purescript/purescript-prelude/releases/tag/v2.0.0) - 2016-09-26
+
+**Breaking changes**
+- Fixed `=<<` to be right associative, as originally intended
+- Changed behaviour of `Eq` and `Ord` for `NaN` numeric value #91 (@berdario)
```

Comment:
Unfortunately, unlike the GitHub releases, the raw CHANGELOG readme won't link things like `#91` for pull requests and `@berdario` for GitHub user profiles.

We can choose to fix these (increasing the workload) or leave them as-is (if required, folks can go look up the pull request or user profile themselves) or decide not to track them (removing these references -- I think this would be losing valuable information).

My gut sense is that we probably should accept that changelogs pre-0.14 should be treated as-is, and we simply try to use the standard format (breaking changes / new features / bugfixes / other) with proper links going forward.

Reply:
FWIW in Spago we use the same format, and I think it's valuable to track this info even if it's not directly linking. I think there's actually value in leaving such links out, because it makes maintaining the changelog less burdensome (I'd say lowering the treshold to keep this up to date is key here), and the raw file (i.e. when you're not looking at a markdown render) stays readable.